packages/core/src/addresses/types.ts

```diff
@@ -62,7 +62,7 @@ export type Bridges = {
 }
 
 export type Bonders = {
-  [token: string]: string[]
+  [token: string]: Record<string, Record<string, string>>
```

Comment:
Up for discussion, as I don't have too strong of an opinion here.

It might make sense to explicitly lay out the structure here. Things look good now, but I could see an annoying bug in the future where copy/paste goes wrong and a `source` doesn't have the correct `destination`s. For example, we could accidentally do:

```
polygon: {
  ethereum: ...
  optimism: ...
  arbitrum: ...
  polygon: ... // <------------ supposed to be xDai
}
```

So maybe it is something like

```
type PolygonRoute = {
  ethereum: ...
  optimism: ...
  arbitrum: ...
  xdai: ...
}

...

type Routes = {
  PolygonRoutes,
  OptimismRoutes,
  ...
}


export type Bonders = {
  [token: string]: Routes
```

Reply:
Resolved by https://github.com/hop-protocol/hop/commit/31b6b58f9d3edd92481619bfbd066287e0b74d94